src/code/code_80122660.c

```diff
@@ -1,5 +1,5 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_80122660/func_80122660.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_pause/func_80122660.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_80122660/func_80122670.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_pause/func_80122670.s")
```

Comment:
I see you made a z_pause file, so you should delete this file to avoid duplication.